### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mitko1717**
i added my greeting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1631,4 +1631,11 @@ Glad to join and good luck, kottans!!!
 Come on, let me see what is happening here!
 ![cat in glasses](https://media4.giphy.com/media/VbnUQpnihPSIgIXuZv/giphy-downsized.gif)
 +
----
+
+
+## Hi everyone!
+
+I'm glad I found ur course!
```
Comment (Temu4) [inline README.md:1638]: you probably want to write `our`?

**Review by Temu4 — CHANGES_REQUESTED** [review]
Hello!
Good job! 🐱 
But can you get the latest changes and fix conflicts?

**Comment by babayK0** [comment]
@mitko1717 Please, fix conflicts

**Comment by lenadgit** [comment]
How to resolve conflicts 
https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line

**Comment by babayK0** [comment]
@mitko1717 Fix conflicts, and i will approve :smiley:

**Comment by mitko1717** [comment]
i really don't understand what conflicts u r talking about


чт, 29 окт. 2020 г. в 13:58, Bohdan Kovalchuk <notifications@github.com>:

**Comment by babayK0** [comment]
@mitko1717 you have conflicts in README.md

https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts

**Comment by babayK0** [comment]
You need pull changes from remote, resolve conflicts and push your code again.

**Comment by Temu4** [comment]
@mitko1717 since that time when you've forked origin repo and added your changes some people made their own changes and pushed them.
So you can just pull the latest updates from the original repo and fix conflicts.
If you need more info, plz, fill free to touch me 😸

**Comment by mitko1717** [comment]
hi. today i ended one more stage and wanted to make push from my local to
add the folder with screenshots, but something went wrong.
i got this message from my terminal:

git push

To https://github.com/mitko1717/kottans-frontend/

 ! [rejected]        main -> main (non-fast-forward)

error: failed to push some refs to '
https://github.com/mitko1717/kottans-frontend/'

hint: Updates were rejected because the tip of your current branch is behind

hint: its remote counterpart. Integrate the remote changes (e.g.

hint: 'git pull ...') before pushing again.

hint: See the 'Note about fast-forwards' in 'git push --help' for details.


then i tried to pull from remote, but

git pull

error: Your local changes to the following files would be overwritten by
merge:

  mock-repo

then i tried to make pull of the forked repo (mock-repo)

i got this:

$ git pull https://github.com/mitko1717/mock-repo

warning: no common commits

remote: Enumerating objects: 2553, done.

remote: Total 2553 (delta 0), reused 0 (delta 0), pack-reused 2553

Receiving objects: 100% (2553/2553), 167.40 MiB | 874.00 KiB/s, done.

Resolving deltas: 100% (1181/1181), done.

From https://github.com/mitko1717/mock-repo

 * branch            HEAD       -> FETCH_HEAD

fatal: refusing to merge unrelated histories


what's wrong? plz help me. ive already pushed and pulled some files.
everything was good

вс, 1 нояб. 2020 г. в 21:32, Artem Vorobiov <notifications@github.com>:

**Review by babayK0 — APPROVED** [review]
:+1:

**Pull request merged**